toolchain/driver/clang_runner.h

```diff
@@ -49,6 +49,22 @@ class ClangRunner {
   // Run Clang with the provided arguments.
   auto Run(llvm::ArrayRef<llvm::StringRef> args) -> bool;
 
+  // Enable leaking memory.
+  //
+  // Clang can avoid deallocating some of its memory to improve compile time.
+  // However, this isn't compatible with library-based invocations. When using
+  // the runner in a context where memory leaks are acceptable, such as from a
+  // command line driver, you can use this to enable that leaking behavior. Note
+  // that this will not override _explicit_ `args` in a run invocation that
+  // cause leaking, it will merely disable Clang's libraries injecting that
+  // behavior.
+  //
+  // Optionally, passing `false` will disable leaking on subsequent runs,
+  // overriding any previous calls.
+  auto EnableLeakingMemory(bool enable = true) -> void {
```

Comment:
The `false` value doesn't seem to be used, is it be worth dropping the argument until we have a use-case?

Reply:
Eh, it just seemed like a cleaner API to me. But ok.